test-setup/scripts/init-psmdb-kerberos.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker exec --user root psmdb-kerberos bash -c 'chown mongodb:root /krb5/mongodb.keytab'
```

Comment:
```suggestion
docker exec --user root psmdb-kerberos bash -c 'chown mongodb:mongodb /krb5/mongodb.keytab'
```

Reply:
Looks like there's no `mongodb` group, only the user.